src/asm/rgbasm.5

```diff
@@ -812,6 +812,66 @@ first, followed by the one from
 and the one from
 .Ql bar.o
 last.
+.Ss Inline Fragments
+Inline fragments are useful for short blocks of code or data that are only referenced once.
+They are section fragments created by surrounding instructions or directives with
+.Ql [[
+double brackets
+.Ql ]] ,
+without a separate
+.Ic SECTION FRAGMENT
+declaration.
+.Pp
+The content of an inline fragment becomes a section fragment, sharing the same constraints as its parent ROM section.
+Its parent section also becomes a section fragment if it is not one already, and RGBLINK merges them together.
+Inline fragments are appended after their parent section in no particular order.
+.Pp
+Any instance of a 16-bit integer constant
+.Ql n16
+in the
+.Xr gbz80 7
+documentation can define an inline fragment, and so can
```

Comment:
I'd rather word it as "inline fragments are to be used in expressions [possibly insert a `.Sx` here], and they evaluate to their starting address."

Well, that's if they are allowed in any expression, I suppose; but I still think such wording would be clearer.

Reply:
The wording is updated; it might need revising further.